artemis/reporting/modules/lfi_detector/reporter.py

```diff
@@ -0,0 +1,59 @@
+import os
+import urllib.parse
+from typing import Any, Callable, Dict, List
+
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.normal_form import NormalForm, get_domain_normal_form
+from artemis.reporting.base.report import Report
+from artemis.reporting.base.report_type import ReportType
+from artemis.reporting.base.reporter import Reporter
+from artemis.reporting.base.templating import ReportEmailTemplateFragment
+from artemis.reporting.utils import get_top_level_target
+
+
+class LFIDetectorReporter(Reporter):
+    LFI_CORE = ReportType("lfi:core")
+
+    @staticmethod
+    def create_reports(task_result: Dict[str, Any], language: Language) -> List[Report]:
+        if task_result["headers"]["receiver"] != "lfi_detector":
+            return []
+
+        if not task_result["status"] == "INTERESTING":
+            return []
+
+        if not isinstance(task_result["result"], dict):
+            return []
+
+        url_parsed = urllib.parse.urlparse(task_result["result"]["result"][0]["url"])
+
+        return [
+            Report(
+                top_level_target=get_top_level_target(task_result),
+                target=f"{url_parsed.scheme}://{url_parsed.netloc}",
+                report_type=LFIDetectorReporter.LFI_CORE,
```

Comment:
_CORE is not needed here - for sql_injection_detector we've added that to distinguish from sql_injection from sqlmap, but here we have only one LFI module